src/main/java/com/github/distributedtracing/ContinuationLocal.java

```diff
@@ -0,0 +1,170 @@
+package com.github.distributedtracing;
```

Comment:
I do +1 it myself.
org.continuation is a good namespace, since it's not directly tied to distributed tracing. One could use a ContinuationLocal for other things as well, like passing around business logic state, even though I would certainly not recommend that.